### PR TITLE
Add new themes to `ColorTable`

### LINKED ISSUE #310: New color themes?
This request adds new color themes to the ColorTable class in PrettyTable. This provides users with built in options for customizing the appearance of their tables.

Changes made:
- Added 8 new themes to the `Themes` class in `colortable.py`:
  - FOREST
  - SUNSET
  - OCEAN_DEEP
  - LAVENDER
  - MONOCHROME
  - NEON
  - EARTH
  - PASTEL

Benefits:
1. Increased customization options for users
2. More variety in color schemes to suit different preferences and use cases
3. Includes themes with both basic and extended color palettes for compatibility with various terminal environments

Testing:
- Tested all new themes in multiple terminal environments to ensure compatibility
- Verified that existing functionality remains unchanged

Usage example:
Screenshot 2024-09-22 at 11 17 24 PM
Screenshot 2024-09-22 at 11 17 33 PM
Screenshot 2024-09-22 at 11 17 41 PM

### PR DESCRIPTION
**Pull request by Bikes4Fun**
Fixes #310.

### CODE REVIEW

File: tests/test_colortable.py [hunk 1]
```diff
@@ -155,3 +155,19 @@ def test_color_table_rendering(self) -> None:
...
+        )
+        table.title = "Theme Test"
+        table.add_row([1, 2, 3, 4, 5, 6])
+
+        for theme_name, theme in vars(Themes).items():
+            if isinstance(theme, Theme):
+                table.theme = theme
+                result = str(table)
+                print(f"\n{theme_name} Theme:")
+                print(result)
```
Comment (hugovk) [inline tests/test_colortable.py]: Let's remove printing from the tests:
```suggestion
```

Comment (Bikes4Fun) [reply tests/test_colortable.py]: Ooops. Yep I'll do that today!

Comment (hugovk) [reply tests/test_colortable.py]: Thanks!

Tip: you can also click the "Commit suggestion" button here in the GitHub UI.

**Comment by hugovk** [comment]
For future reference, here's the existing two:

image

And the new ones:

image

Thank you for your contribution!

File: tests/test_colortable.py (hunk 1 above)
Comment (hugovk) [reply tests/test_colortable.py]: I clicked the button so I can include it in the next release. Thanks again!

**Pull request merged**